plutus-core/plutus-core/src/PlutusCore/Evaluation/ErrorWithCause.hs

```diff
@@ -54,7 +54,7 @@ instance (PrettyBy config cause, PrettyBy config err) =>
 
 instance (PrettyPlc cause, PrettyPlc err) =>
             Show (ErrorWithCause err cause) where
-    show = render . prettyPlcReadableSimple
+    show = render . prettyPlcReadable
```

Comment:
Not sure how useful this is, but I think it's potentially more useful than noisy.